app/models/thredded/user_topic_follow.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+module Thredded
+  class UserTopicFollow < ActiveRecord::Base
+    REASON_MANUAL = 'manual'
```

Comment:
How about making this an enum? http://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html